runtime/runtime.py

```diff
@@ -523,6 +575,7 @@ def send_motor_data(data):
     })
 
     # Update sensor values, and send to UI
+    update_hibike()
```

Comment:
Are we still planning to add hot plugging for motors? We don't have that much time left lol